libshaderc/src/shaderc_cpp_test.cc

```diff
@@ -290,6 +306,44 @@ TEST_F(CppInterface, DisassemblyOption) {
   EXPECT_THAT(result_from_cloned_options.GetData(), HasSubstr("MemoryModel"));
 }
 
+TEST_F(CppInterface, GenerateDebugInfoBinary) {
+  options_.SetGenerateDebugInfo();
+  // The output binary should contain the name of the vector: debug_info_sample as char
+  // array.
+  EXPECT_THAT(CompilationOutput(kMinimalDebugInfoShader,
+                                shaderc_glsl_vertex_shader, options_),
+              HasSubstr("debug_info_sample"));
```

Comment:
I think we may have two Compilation**Output for binary and disassembly output respectively. But how to use the 'MakeVector' in Spirv-tools' test? It is an internal header, I think I shouldn't refer to that header, maybe I should copy the function to shaderc's tests? It is only a utility function depends only on STL.

Then, if I use the spvBinaryParse(), we will need to add SPIRV-Tools as a dependent library for shaderc's test, and only for the test. Is this what you mean, or I missed something?